### DISCUSSION TITLE
I have a problem when use Jest, MSW with Vite. I don't get mock for Axios.

### DISCUSSION THREAD

**Discussion by THUONGBUIANH** [mswjs/msw#1629] [Q&A]
### Prerequisites

- [X] I confirm my issue is not in the [opened issues](https://github.com/mswjs/msw/issues)
- [X] I confirm the [Frequently Asked Questions](https://mswjs.io/docs/faq) didn't contain the answer to my issue

### Environment check

- [X] I'm using the [latest](https://github.com/mswjs/msw/releases/latest) `msw` version
- [X] I'm using Node.js version 14 or higher

### Node.js version

v16.14.2

### Reproduction repository

https://github.com/THUONGBUIANH/React-testing-library-sample

### Reproduction steps

![image](https://github.com/mswjs/msw/assets/37923990/ec5f55c6-eda1-4a4d-a4b7-3fb7bb707cb4)
![image](https://github.com/mswjs/msw/assets/37923990/033c5f12-62e0-4467-9552-c45c28d8f279)
![image](https://github.com/mswjs/msw/assets/37923990/458f34d8-a4d1-47ea-9146-550aff01d2e8)
![image](https://github.com/mswjs/msw/assets/37923990/525f6850-afd9-4d69-b15a-57b1571292c6)
![image](https://github.com/mswjs/msw/assets/37923990/4eac7da1-f6fe-4645-921a-1ab15de5150c)

### Current behavior

![image](https://github.com/mswjs/msw/assets/37923990/d440d236-d308-47ca-92ec-2278d237e207)

### Expected behavior

How to solve and mock response for axios? 
And How to show console response of Mock Axios?

**Comment by THUONGBUIANH** [comment]
Please, help me, thank you!

**Comment by kettanaito** [comment]
Hi.

It seems that the mocked response you're returning has incorrect shape:

https://github.com/THUONGBUIANH/React-testing-library-sample/blob/c3ef2b0f351a8b61f838657e924f7916ab6b8764/src/mock/handlers.ts#L7

Drop the `data` nesting here. The `data` in Axios is a fixed property that exposes you the response as-is. 

https://github.com/THUONGBUIANH/React-testing-library-sample/blob/c3ef2b0f351a8b61f838657e924f7916ab6b8764/src/components/FollowersList/FollowersList.tsx#L12

What you have now yields `data.data.results`, which is not what you expect in code.

**Reply by THUONGBUIANH** [reply]
Thank you very much. It's help for me. 
However, I would like to console log the response mock Axios when I run test. 
Please, could you share me some tips for see response mock Axios when I run the test?
